### ISSUE TITLE
Add support for gp3 disk type

### ISSUE THREAD

**Issue by tewfik-ghariani** [NixOS/nixops-aws#131]
The gp3 volumes provide better performance (independently of the volumes size) and are cheaper in terms of storage pricing.

```
Volume                                          TypePrice
General Purpose SSD (gp3) - Storage             $0.08/GB-month
General Purpose SSD (gp3) - IOPS                3,000 IOPS free and $0.005/provisioned IOPS-month over 3,000
General Purpose SSD (gp3) - Throughput          125 MB/s free and $0.04/provisioned MB/s-month over 125
General Purpose SSD (gp2) Volumes               $0.10 per GB-month of provisioned storage
```

Ref: https://aws.amazon.com/blogs/aws/new-amazon-ebs-gp3-volume-lets-you-provision-performance-separate-from-capacity-and-offers-20-lower-price

Suggested by @Bsami

**Comment by lionello** [comment]
Fixed by #135

**Comment by tewfik-ghariani** [comment]
Thanks!
